base_layer/core/src/types.rs

```diff
@@ -78,11 +78,14 @@ pub const MAX_RANGE_PROOF_RANGE: usize = 64; // 2^64
 
 /// Current version of the blockchain
 pub const BLOCKCHAIN_VERSION: u16 = 0;
+
+// ToDo move this to some other consensus like struct to be passed around. cfg test is not set during integration tests.
```

Comment:
Yes, I was thinking exactly the same thing

Reply:
#643